.github/ISSUE_TEMPLATE/iceberg_bug_report.yml

```diff
@@ -76,3 +76,11 @@ body:
         You can include files by dragging and dropping them here.
     validations:
       required: true
+  - type: checkboxes
+    attributes:
+      label: Willingness to contribute
+      description: The Iceberg community encourages bug fix contributions. Would you or another member of your organization be willing to contribute a fix for this bug to the Iceberg codebase?
```

Comment:
nit: I would rather use Apache Iceberg instead of Iceberg.

Reply:
updated